docs/src/Tabs.doc.js

```diff
@@ -31,11 +31,18 @@ card(
       },
       {
         name: 'onChange',
-        type: `({ event: SyntheticMouseEvent<>, activeTabIndex: number }) => void`,
+        type: `({ +event: SyntheticMouseEvent<> | SyntheticKeyboardEvent<>, +activeTabIndex: number }) => void`,
         required: true,
         description:
           'If your app uses a tool such as react-router to navigate between pages, be sure to use onChange to navigate instead of getting a full page refresh with href',
       },
+      {
+        name: 'gap',
```

Comment:
@jackhsu978 just checked in with Annie Teng & @ashleyseto and both of them mentioned they would rather not have the extra gap in between.

Could you also follow up with them on the font size change?

Reply:
Yeah, according to @ajmoo Button's size also doesn't influence the text size within. I think it's good to be consistent, so I changed the logic back.